test-app/frontend/src/App/App.tsx

```diff
@@ -15,13 +17,16 @@ function Private() {
 }
 
 function App() {
+  // TODO-rtotale: Submit personal info
+  const onSubmitPersonalInfo = (data: object) => console.log(data);
   return (
     <UserProvider>
       <Authenticator.Provider>
         <BrowserRouter>
           <Routes>
             <Route path="/" element={<StandardPage component={<Home />} />} />
             <Route path="/forms" element={<StandardPage component={<FormsViewer forms={MOCK_FORMS} />} />} />
+            <Route path="/personal-info" element={<StandardPage component={<CustomForm schema={personalInfoSchema} onSubmit={onSubmitPersonalInfo}/>} />} />
```

Comment:
Would like to see the personal info stuff out into it's own separate component I think in case future uses means it needs more than just a form. Also don't love seeing the onSubmit in the app component, which could be refactored out into the new component.